Update guide.md with the scale up information (#1153)
Azure-Samples/cognitive-services-speech-sdk: samples/ingestion/ingestion-client/Setup/guide.md

* **Only .wav PCM is supported.**
* **You will need to use the [Real Time ARM Template](ArmTemplateRealtime.json) from the repository for this operating mode.**

#### Scale up

> Batch mode will process transcription requests following best effort policies via which compute is requested as soon as the transcription is scheduled. Available compute is directly allocated. 

Reviewer: ```suggestion
Batch mode will process transcription requests following best effort policies using the compute you request when the transcription is scheduled. Available compute is directly allocated. 
```

Is my change an improvement? I'm not sure!

> In Real time mode, each Azure Speech resource is allocated by default 50 concurrent connections, which indicates the maximum number of parallel audio transcription streams. Customers do have a choice to raise a request to have that number increased through Azure Speech customer channels. To avoid throttling however the rythm/tempo of uploading audio files to Azure storage (therefore triggering real time transcriptions) should be controlled. If it is not contolled it will result in throttling.

Reviewer 2: typo "contolled"

Reviewer 2: typo "rhythm"

Author: addressed. thank you.

> In Real time mode, each Azure Speech resource is allocated by default 50 concurrent connections, which indicates the maximum number of parallel audio transcription streams. Customers do have a choice to raise a request to have that number increased through Azure Speech customer channels. To avoid throttling however the rhythm/tempo of uploading audio files to Azure storage (therefore triggering real time transcriptions) should be controlled. If it is not controlled it will result in throttling.

Reviewer: ```suggestion
In Real time mode, each Azure Speech resource is allocated with a default of 50 concurrent connections, which indicates the maximum number of parallel audio transcription streams. Customers can request a higher limit. To avoid throttling, the cadence of new audio file uploads to Azure storage should be controlled, because each upload triggers  a real-time transcription. Throttling occurs when the concurrency limit is reached.
```
